esr21_reports/views/psrt_mixins/demographics_mixin.py

```diff
@@ -0,0 +1,362 @@
+
+            totals = self._get_age_range(site_id, 30, 39)
+            age_30_to_40.append(totals)
+
+            totals = self._get_age_range(site_id, 40, 49)
+            age_40_to_50.append(totals)
+
+            totals = self._get_age_range(site_id, 50, 59)
+            age_50_to_60.append(totals)
+
+            totals = self._get_age_range(site_id, 60, 69)
+            age_60_to_70.append(totals)
+
+            totals = self._get_age_range(site_id, 70)
+            age_70_and_above.append(totals)
+
+        age_18_to_30.insert(0, sum(age_18_to_30))
+        age_30_to_40.insert(0, sum(age_30_to_40))
+        age_40_to_50.insert(0, sum(age_40_to_50))
+        age_50_to_60.insert(0, sum(age_50_to_60))
+        age_60_to_70.insert(0, sum(age_60_to_70))
+        age_70_and_above.insert(0, sum(age_70_and_above))
+
+        return dict(age_18_to_30=age_18_to_30, 
+                    age_30_to_40=age_30_to_40, 
+                    age_40_to_50=age_40_to_50, 
+                    age_50_to_60=age_50_to_60, 
+                    age_60_to_70=age_60_to_70,
+                    age_70_and_above = age_70_and_above)
+
+
+    @property
+    def hiv_statistics(self):
+        hiv_positive = []
+        hiv_negative = []
+        hiv_unknown = []
+
+
+        for site_id in self.site_ids:
+
+            status = POS
+            positive = RapidHIVTesting.objects.filter((Q(subject_visit__subject_identifier__in=self.enrolled_pids) & Q(
+                site_id=site_id)) & (Q(hiv_result=status) | Q(rapid_test_result=status))).count()
+
+            status = NEG
+            negative = RapidHIVTesting.objects.filter((Q(subject_visit__subject_identifier__in=self.enrolled_pids) & Q(
+                site_id=site_id)) & (Q(hiv_result=status) | Q(rapid_test_result=status))).count()
+
+            status = IND
+            unknown = RapidHIVTesting.objects.filter((Q(subject_visit__subject_identifier__in=self.enrolled_pids) & Q(
+                site_id=site_id)) & (Q(hiv_result=status) | Q(rapid_test_result=status))).count()
+
+            hiv_positive.append(positive)
+            hiv_negative.append(negative)
+            hiv_unknown.append(unknown)
+
+
+        hiv_positive.insert(0, sum(hiv_positive))
+        hiv_negative.insert(0, sum(hiv_negative))
+        hiv_unknown.insert(0, sum(hiv_unknown))
+
+        return [['HIV Positive', *hiv_positive], 
+                ['HIV Negative', *hiv_negative],
+                ['Unknown', *hiv_unknown]]
+
+    @property
+    def smoking_statistics(self):
+        never_smoked = []
+        occasional_smoker = []
+        current_smoking = []
+        previous_smoker = []
+
+
+        for site_id in self.site_ids:
+
+            never_smoked_count = MedicalHistory.objects.filter(site_id=site_id, subject_visit__subject_identifier__in=self.enrolled_pids,
+                                                        smoking_status='never_smoked').values_list('subject_visit__subject_identifier').distinct().count()
```

Comment:
fix you pep8, lines too long
install pep8 or flake8
run command pep8

Reply:
formatted the whole file